### PR TITLE
Update Openstack dynamic inventory reference

### PR DESCRIPTION
**Pull request by marsavela**
+label: docsite_pr

##### SUMMARY
This MR updates the name of the file that creates the dynamic inventory on Openstack, since it was outdated.

##### ISSUE TYPE
 - Docs Pull Request

### CODE REVIEW

**Comment by Akasurde** [comment]
@czunker Could you please review this so that we can proceed ?

**Review by czunker — APPROVED** [review]
Looks good to me

**Comment by czunker** [comment]
@Akasurde Done.

Sorry, didn't see this one during my search for the bug.

**Comment by Akasurde** [comment]
@czunker No problem. Will wait for @emonty to comment and then we can proceed.

**Comment by Akasurde** [comment]
@marsavela Could you please rebase this and let me know ? Also, add additional line from #41563.

**Comment by marsavela** [comment]
There is also #42323 as part of this. I rebased and updated this PR with all the changes together.

**Comment by marsavela** [comment]
Forgot to ping @Akasurde

**Comment by marsavela** [comment]
@Akasurde @gundalow I rebased once again. Please review. @czunker Would be great if you can confirm the changes once again.

**Review by rcarrillocruz — APPROVED** [review]

**Review by Akasurde — APPROVED** [review]
shipit

File: docs/docsite/rst/user_guide/intro_dynamic_inventory.rst [hunk 1]
```diff
@@ -246,19 +246,22 @@ explicitly clear the cache, you can run the ec2.py script with the ``--refresh-c
...
 
 Download the latest version of the OpenStack dynamic inventory script and make it executable::
 
-    wget https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack.py
-    chmod +x openstack.py
+    wget https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack_inventory.py
+    chmod +x openstack_inventory.py
+
+.. note::
+    Do not name it `openstack.py`. This name will conflict with imports from openstacksdk.
```
Comment (gundalow) [inline docs/docsite/rst/user_guide/intro_dynamic_inventory.rst:264]: Thanks for adding this :)

**Review by gundalow — APPROVED** [review]

File: docs/docsite/rst/user_guide/intro_dynamic_inventory.rst (hunk 1 above)
Comment (marsavela) [reply docs/docsite/rst/user_guide/intro_dynamic_inventory.rst:264]: Cherry-picked @czunker's PR. (Y)

**Review by czunker — APPROVED** [review]
Thanks

**Pull request merged**